charts/stable/pod-gateway/templates/webhook-deployment.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: {{ include "common.capabilities.deployment.apiVersion" . }}
+apiVersion: apps/v1
```

Comment:
What made you change this?

Reply:
It was changed in the Deployment template in common as well.